script.sinaweibo/resources/skins/default/720P/script-sinaweibo-Twitter.xml

```diff
@@ -0,0 +1,286 @@
+						<font>font14</font>
+						<align>left</align>
+						<textcolor>cadetblue</textcolor>
+						<shadowcolor>AA000000</shadowcolor>
+					</control>
+					<control type="label">
+						<description>Tweet timedelta</description>
+						<posx>720</posx>
+						<posy>5</posy>
+						<width>400</width>
+						<height>70</height>
+						<label>$INFO[ListItem.Property(timedelta)]</label>
+						<font>font12</font>
+						<align>right</align>
+						<textcolor>cadetblue</textcolor>
+						<shadowcolor>AA000000</shadowcolor>
+					</control>
+					<control type="textbox">
+						<description>Tweet Text</description>
+						<posx>95</posx>
+						<posy>30</posy>
+						<width>1000</width>
+						<height>70</height>
+						<label>$INFO[ListItem.Label]</label>
+						<font>font14</font>
+						<align>left</align>
+						<textcolor>white</textcolor>
+						<shadowcolor>AA000000</shadowcolor>
+						<pulseonselect></pulseonselect>
+					</control>
+				</focusedlayout>
+			</control>
+			<control type="scrollbar" id="61">
+				<left>1082</left>
+				<top>85</top>
+				<width>15</width>
+				<height>540</height>
+				<texturesliderbackground colordiffuse="29FFFFFF">white.png</texturesliderbackground>
+				<texturesliderbar colordiffuse="AAFFFFFF">white.png</texturesliderbar>
+				<texturesliderbarfocus colordiffuse="FF12B2E7">white.png</texturesliderbarfocus>
+				<textureslidernib>-</textureslidernib>
+				<textureslidernibfocus>-</textureslidernibfocus>
+				<showonepage>false</showonepage>
+				<orientation>vertical</orientation>
+				<onleft>32501</onleft>
+				<onright>32501</onright>
+				<ondown>61</ondown>
+				<onup>61</onup>
+				<animation effect="slide" end="120,0" time="0" condition="!Control.IsVisible(5)">Conditional</animation>
+			</control>
+		</control>
+		<control type="button" id="32514">
+			<description>Reset</description>
+			<left>550</left>
+			<top>630</top>
+			<width>200</width>
+			<height>80</height>
+			<label>$ADDON[script.sinaweibo 32042]</label>
+			<texturefocus border="30" colordiffuse="FF12B2E7">button-fo.png</texturefocus>
+			<texturenofocus border="30">button-nofo.png</texturenofocus>
+			<font>font13</font>
+			<align>center</align>
+			<aligny>center</aligny>
+			<textcolor>white</textcolor>
+			<disabledcolor>disabled</disabledcolor>
+			<textoffsetx>7</textoffsetx>
+			<pulseonselect>no</pulseonselect>
+			<onleft>32501</onleft>
+			<onright>32501</onright>
+			<onup>32501</onup>
+			<ondown>32501</ondown>
+		</control>
+		<!--Busy control-->
+		<control type="image" id="32540">
+			<description>Busy animation</description>
+			<posx>450</posx>
+			<posy>300</posy>
+			<width>120</width>
+			<height>120</height>
+			<texture></texture>
+			<aspectratio>keep</aspectratio>
+			<animation effect="rotate" start="0" end="360" center="510,360" time="1200" loop="true" condition="true">conditional</animation>
+			<visible>!IsEmpty(Window(home).Property(loading-script-matchcenter-twitter))</visible>
```

Comment:
Carefull with property names. Also  I doubt this will work on v18, you might want to test